forms-flow-web/src/components/Form/Stepper.js

```diff
@@ -415,10 +415,11 @@ const mapStateToProps = (state) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    getAllProcesses: () => {
+    getAllProcesses: (tenantKey) => {
+      console.log("tenatn",tenantKey);
```

Comment:
Remove console

Reply:
Check for Multitenancy is enabled else sent null for tenantKey

Reply:
null is already set as default